### PR TITLE
dbuild : Add smartfs command support

### PR DESCRIPTION
**Pull request by YiHyunJin**
Docker version modify for docker update. And add command.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d70d0cd7873847125a699592cc792ed2f590f9cc] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d70d0cd7873847125a699592cc792ed2f590f9cc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [06bad416648c2b857ddfa4439b0ac9cc6dc0eb55] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [06bad416648c2b857ddfa4439b0ac9cc6dc0eb55] - Code Rule Check (C++) OK.

File: tools/ramdump/dumpParser.py [hunk 1]
```diff
@@ -619,7 +619,11 @@ def main():
 		g_stext = rParser.get_address_of_symbol("_stext")
 		global g_etext
 		g_etext = rParser.get_address_of_symbol("_etext")
+		global g_mmheap
+		g_mmheap = rParser.get_address_of_symbol("g_mmheap")
 
+		print "@@@@@@@@@@@@ g_mmheap : ", hex(g_mmheap)
```
Comment (sunghan-chang) [inline tools/ramdump/dumpParser.py]: Check this file

**Comment by seinfra** [comment]
Target : [505e6285af113e36b32d0c6aa98e5913ca7554df] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [505e6285af113e36b32d0c6aa98e5913ca7554df] - Code Rule Check (C++) OK.

**Pull request merged**